### PR TITLE
Removed references to make docs

### PR DESCRIPTION
**Pull request by GSonderling**
#13682  

I removed the leftover reference as requested.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -495,8 +493,6 @@ help:
 	@echo 'to check the official mods for erroneous yaml files, run:'
 	@echo '  make test'
 	@echo
-	@echo 'to generate documentation aimed at modders, run:'
-	@echo '  make docs'
 	@echo
```
Comment (pchote) [inline Makefile]: Can you please remove one of these empty lines too? Otherwise there is two between 'make test' and 'to install, run'.

**Review by pchote — COMMENTED** [review]
Just one minor nit, then LGTM

File: Makefile (hunk 1 above)
Comment (GSonderling) [inline Makefile]: You mean one of the empty `@echo` on lines 495 and 496 correct?

File: Makefile [hunk 2]
```diff
@@ -14,8 +14,6 @@
 # to check the official mod dlls for StyleCop violations, run:
 #   make check
 #
-# to generate documentation aimed at modders, run:
-#   make docs
 #
```
Comment (obrakmann) [inline Makefile]: Sorry, this is really nit-picky, but can you remove one of these two empty lines as well, please?

**Review by obrakmann — APPROVED** [review]
👍 after that

**Pull request merged**